bin/tictactoe

```diff
@@ -1,3 +1,33 @@
 #!/usr/bin/env ruby
 
 require_relative '../config/environment'
+
+puts "Let's play Tic Tac Toe!" 
+puts "Please enter 1 or 2 players, or enter 0 for if you just want to watch."
+count = gets.chomp
+
+ 
+case count
+  
+  when "0"
+    first_player = Player::Computer.new("X") 
+    second_player = Player::Computer.new("O")
+  when "1"
+    puts "Enter 1 - 'X' - to go first or 2 - 'O' - for the computer to go first."
+    input = gets.chomp
+    if input == "1"
+      first_player = Player::Human.new("X")
+      second_player = Player::Computer.new("O") 
+    elsif input == "2"
+      first_player = Player::Computer.new("X")
+      second_player = Player::Human.new("O")
+    end
+  when "2"
+    first_player = Player::Human.new("X")
+    second_player = Player::Human.new("O")
+  else
+    puts "Guess not? Maybe next time!"
+
+end
+
+Game.new(first_player, second_player).play
```

Comment:
Prefer to encapsulate CLI logic into a class. For example, the only code in your bin executable should be:

```ruby
#!/usr/bin/env ruby
       
require_relative '../config/environment'

TicTacToeCLI.new.call
```

And then you'd have a file: `lib/tic_tac_toe_cli.rb` with:

```ruby
class TicTacToeCLI
  def initialize
  end

  def call
   # Your game logic
  end
end
```

The code in your bin would generally just go in `call`.